README.md

```diff
@@ -31,85 +31,93 @@ The service is subscribed to the following [Keptn CloudEvents](https://github.co
+```console
+kubectl -n keptn create secret generic dynatrace \
+--from-literal="DT_API_TOKEN=$DT_API_TOKEN" \
+--from-literal="DT_TENANT=$DT_TENANT" \
+--from-literal="KEPTN_API_URL=$KEPTN_API_URL" \
+--from-literal="KEPTN_API_TOKEN=$KEPTN_API_TOKEN" \
+--from-literal="KEPTN_BRIDGE_URL=$KEPTN_BRIDGE_URL" \
+-oyaml --dry-run=client | kubectl replace -f -
+```
 
+ If the Keptn credentials are omitted from this main secret, `KEPTN_API_TOKEN` must be provided by the `keptn-api-token` secret. Furthermore, `dynatraceService.config.keptnApiUrl` and optionally `dynatraceService.config.keptnBridgeUrl` must be set when applying the helm chart (see below).
 
-    ```console
-    helm upgrade --install dynatrace-service -n keptn https://github.com/keptn-contrib/dynatrace-service/releases/download/$VERSION/dynatrace-service.tgz --set dynatraceService.config.httpProxy=http://mylocalproxy:1234 --set dynatraceService.config.httpsProxy=https://mylocalproxy:1234
-    ```
+### 3. Deploy the Service
 
-* To deploy the current version of the *dynatrace-service* in your Kubernetes cluster, use the helm chart located in the `chart` directory.
+To deploy the current version of the *dynatrace-service* in your Kubernetes cluster, use the helm chart located in the `chart` directory.
 Please use the same namespace for the *dynatrace-service* as you are using for Keptn, e.g: keptn.
 
-    ```console
-    helm upgrade --install dynatrace-service -n keptn https://github.com/keptn-contrib/dynatrace-service/releases/download/$VERSION/dynatrace-service-$VERSION.tgz
-    ```
-   
-   **Note**: Replace `$VERSION` with the desired version number (e.g., 0.13.0) you want to install.
-   
-   This installs the `dynatrace-service` in the `keptn` namespace, which you can verify using:
+```console
+helm upgrade --install dynatrace-service -n keptn https://github.com/keptn-contrib/dynatrace-service/releases/download/$VERSION/dynatrace-service-$VERSION.tgz
+```
+
+The installation can then be verified using:
+
+```console
+kubectl -n keptn get deployment dynatrace-service -o wide
+kubectl -n keptn get pods -l run=dynatrace-service
+```
+
+**Notes**: 
+* Replace `$VERSION` with the desired version number (e.g. 0.15.0) you want to install.
+* Variables may be set by appending key-value pairs with the syntax `--set key=value`
```

Comment:
I think a table with possible values that can be set makes sense here, rather than having it written verbosely using bullet points.
Something like https://github.com/keptn-contrib/dynatrace-service/blob/master/chart/README.md but only the ones that are really relevant for the user (like the ones we mention below).